thirdeye/thirdeye-frontend/app/pods/components/anomaly-id/component.js

```diff
@@ -24,7 +24,7 @@ export default Component.extend({
    * @method anomalyChangeRate
    * @return {Number} - total % change from baseline
    */
-  anomalyChangeRate: computed('anomaly.current', 'anomaly.baseline', function() {
+  anomalyChangeRate: computed('anomaly.{current,baseline}', function() {
```

Comment:
This is interesting to combined props of the same parent object.

Reply:
This is actually the new syntax, ember https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/use-brace-expansion.md